class/Defaults/Trigger/WordPress/ActivePlugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * WordPress Active Plugin trigger
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\WordPress;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+use BracketSpace\Notification\Abstracts;
+
+/**
+ * Active Plugin trigger class
+ */
+class ActivePlugin extends Abstracts\Trigger{
+
+	/**
+	 * Constructor
+	 */
+	public function __construct(){
+
+		parent::__construct( 'wordpress/active_plugin',  __( 'Plugin Active', 'notification' ) );
+
+		$this->add_action( 'activated_plugin', 10, 2 );
+		$this->set_group( __( 'WordPress', 'notification' ) );
+		$this->set_description( __( 'Fires when plugin is active', 'notification' ) );
+
+	}
+
+	/**
+	 * Gets specific update type
+	 *
+	 * @param  string $plugin PluginPath.
+	 * @return void
+	 */
+	public function action($plugin)
```

Comment:
Method params should have space in the parenthesis, ie. `action( $plugin )`
Also, we need some better variable names, because `$plugin` states for something like object or full array which describes the plugin. This is just the plugin relative path, so would be better to name it like `$plugin_rel_path`